language/diem-framework/modules/0L/Testnet.move

```diff
@@ -13,8 +13,12 @@ module Testnet {
 
     struct IsTestnet has key { }
 
+    // Todo: never used, is this ok?
```

Comment:
@0o-de-lally FYI.

Reply:
I think this is called externally, for example by test harnesses